addons/website_slides/controllers/main.py

```diff
@@ -773,7 +773,7 @@ def get_html_content(self, slide_id):
         if fetch_res.get('error'):
             return fetch_res
         return {
-            'html_content': fetch_res['slide'].html_content
+            'html_content': request.env['ir.qweb.field.html'].record_to_html(fetch_res['slide'], 'html_content', {'template_options': {}})
```

Comment:
alternatively, you could do `self.env'ir.qweb']._get_field(record, 'html_content', None, 'div', {}, {}, {})[1]` (copying the values gotten in the non-fullscreen rendering of `<div field="slide.html_content"/>`) but it's not much less hacky so both seems equivalent